packages/jaeger-ui-components/src/TraceTimelineViewer/VirtualizedTraceView.tsx

```diff
@@ -202,12 +202,13 @@ export class UnthemedVirtualizedTraceView extends React.Component<VirtualizedTra
     if (trace !== nextTrace || childrenHiddenIDs !== nextHiddenIDs || detailStates !== nextDetailStates) {
       this.rowStates = nextTrace ? generateRowStates(nextTrace.spans, nextHiddenIDs, nextDetailStates) : [];
     }
-    if (currentViewRangeTime !== nextViewRangeTime) {
+    if (currentViewRangeTime !== nextViewRangeTime || trace !== nextTrace) {
+      console.log('update createViewedBoundsFunc');
```

Comment:
Do we really want to log it there, or was it only for debugging?

Reply:
No I forget to delete it